src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/ActiveDirectoryNativeAuthenticationProvider.cs

```diff
@@ -52,6 +52,7 @@ internal class ActiveDirectoryNativeAuthenticationProvider : SqlAuthenticationPr
             {
                 result = await app.AcquireTokenInteractive(scopes)
                   .WithUseEmbeddedWebView(true)
+                  .WithLoginHint(parameters.UserId)
```

Comment:
WithLoginHint handles an empty userid ok?

Reply:
Yes, it handles null, empty string values, prompts for login in those cases.